scripts/snapshot-restore-balance/snapshot-balances.js

```diff
@@ -72,28 +103,81 @@ async function restoreBalances(api, keyring, snapshotBalancesFile, custodySeed)
     }
 }
 
+function usage() {
+    console.error(`Usage : snapshot or restore
+    node snapshot-balances.js [-e ] snapshot
+        or 
+    node snapshot-balances.js restore "//Alice"
```

Comment:
How about something like the following?
```suggestion
    node snapshot-balances.js restore "//CustodyWalletSeed"
```